importer/parsexml.py

```diff
@@ -0,0 +1,59 @@
+import os
+import xml.etree.ElementTree as ET
+import json
+import sys
+import argparse
+
+
+def main(argv=None):
+    """Parse PubMed XML files in a directory."""
+    argv = argv or sys.argv[1:]
+    parser = argparse.ArgumentParser("Parse XML files.")
+    parser.add_argument("directory",
+                        help="Directory to scan",
+                        default="./Retractions",
+                        type=str,
+                        metavar="DIR")
+    parse_result = parser.parse_args(argv)
+
+    for file in os.listdir(parse_result.directory):
+        if file.endswith(".xml"):
+            tree = ET.parse(parse_result.directory + "/" + file)
+            root = tree.getroot()
+            article_data = {}
+            
+            for medinfo in root.iter('MedlineCitation'):
+                article_data['pmid'] = medinfo.find('PMID').text
+                
+            for author in root.iter('Author'):
+                lastname = author.find('LastName').text
+                firstname = author.find('ForeName').text
+                authorname = firstname + " " + lastname
+                article_data['Author'] = authorname
+
+            for pubDate in root.iter('DateCompleted'):
+                pubYear = pubDate.find('Year').text
+                pubMonth = pubDate.find('Month').text
+                pubDay = pubDate.find('Day').text
+                pubDate = pubDay + "/" + pubMonth + "/" + pubYear
+                article_data['pubDate'] = pubDate
+
+            for reviseDate in root.iter('DateRevised'):
+                reviseYear = reviseDate.find('Year').text
+                reviseMonth = reviseDate.find('Month').text
+                reviseDay = reviseDate.find('Day').text
+                reviseDate = reviseDay + "/" + reviseMonth + "/" + reviseYear
```

Comment:
I think `datetime` has a constructor here that you can use - that way you get an actual serialised date object which can be treated as a real date on the frontend side and inside the database.